application/helpers/update/updates/Update_480.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+class Update_480 extends DatabaseUpdateBase
+{
+    public function up()
+    {
+        $this->db->createCommand()->createTable(
+            '{{sourcemessage}}',
```

Comment:
sourcemessage or source_message?

Reply:
Or message_source??

Reply:
Yiii use SourceMessage , then : what you want ;)

Reply:
Then probable source_message

Sadly we're not consistent in our db naming scheme. But it's supposed to be snake_case.